masonry_core/src/app/render_root.rs

```diff
@@ -170,6 +173,9 @@ pub enum WindowSizePolicy {
 
 /// Options for creating a [`RenderRoot`].
 pub struct RenderRootOptions {
+    /// Default values that Properties will have if not defined per-widget.
```

Comment:
I suspect that `Properties` needn't be capitalized (or it should be a valid, linked identifier).